recipes/restrictedpython/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "restrictedpython" %}
+{% set version = "5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RestrictedPython-{{ version }}.tar.gz
+  sha256: 9ae16e500782b41bd1abefd8554ccb26330817bba9ce090d385aa226f1ca83e8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python <3.10
+  run:
+    - python <3.10
+
+test:
+  imports:
+    - RestrictedPython
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zopefoundation/RestrictedPython
+  summary: RestrictedPython is a defined subset of the Python language which allows to provide a program input into a trusted environment.
+  dev_url: https://github.com/zopefoundation/RestrictedPython
+  license: ZPL-2.1
+  license_file: COPYRIGHT.txt
```

Need this and the license file:
```suggestion
  license_file:
    - COPYRIGHT.txt
    - LICENSE.txt
```